server/routes/prediction.routes.js

```diff
@@ -577,6 +591,9 @@ module.exports = {
     req.body.rows = (req.body.rows !== undefined) ? req.body.rows : 100
     let user = authRoutes.userInfoFromReq(req)
 
+     // Add `userEmail` from request body to the filter
+     req.body.userEmail = req.body.userEmail || user.email;
```

Comment:
You shouldn't need the user email for filtering if the agency & office are already provided

Reply:
All user email filtering has been removed.